### PR TITLE
feat:login-endpiont

### PR DESCRIPTION
**Pull request by EseigbeSamuel**

### CODE REVIEW

**Comment by EseigbeSamuel** [comment]
login endpoint

**Comment by SheyiB** [comment]
> login endpoint

@EseigbeSamuel Great work! I'm impressed! Keep it up!

File: src/services/auth.service.js [hunk 1]
```diff
@@ -1,63 +1,83 @@
...
-        }
+      return customer;
+    } catch (error) {
+      throw error;
     }
+  }
+  async login(data) {
+    try {
+      const { email, password } = body;
+      let user = await this.model.findOne({
```
Comment (SheyiB) [inline src/services/auth.service.js:64]: @EseigbeSamuel can you improve this so we first check for the email and if the email is not found we send a message. After doing that we then check for the password, but remember we created a password match method on customer model that is what we use to check if passwords match. Let me know if you have questions. Great work! Well done

**Review by SheyiB — CHANGES_REQUESTED** [review]

**Review by SheyiB — APPROVED** [review]
Great Work! Super Impressive

**Pull request merged**